src/boat_simulator/boat_simulator/common/constants.py

```diff
@@ -67,3 +80,15 @@ class PhysicsEnginePublisherTopics:
 
 # Max sail actuator control angle range in degrees, min angle [0], max angle [1]
 SAIL_MAX_ANGLE_RANGE = (-7, 7)
+
+# Predetermined values for BoatProperties
+BOAT_PROPERTIES = BoatProperties(
```

Comment:
Can you please add a TODO comment here to mention that these are placeholder values and should be replaced eventually when we have real values

Reply:
alright, will make the changes

Reply:
Added